build/media_source/system/css/fields/calendar.css

```
@@ -98,9 +98,9 @@ div.calendar-container table td.title { /* This holds the current "month, year"
   background-color: var(--calendar-week-bg, #f4f4f4);
 }
 
-.calendar-container table tbody td.day.selected { /* Cell showing today date */
-  color: var(--btn-primary-color);
-  background: var(--btn-primary-bg);
+.calendar-container table tbody td.day.selected {
+  color: var(--btn-primary-color, #fff) !important;
```

```suggestion
  color: var(--btn-primary-color, var(--bs-btn-primary-color, #fff));
```
Explanation: 
if `--btn-primary-color` use it (used in backend template)
else if `--bs-btn-primary-color` use it (default Bootstrap css var)
else drop hardcoded value (non Bootstrap template)

---

> Explanation: if `--btn-primary-color` use it (used in backend template) else if `--bs-btn-primary-color` use it (default Bootstrap css var) else drop hardcoded value (non Bootstrap template)

are you suggesting only these changes ?
you can check once what [changs](https://github.com/joomla/joomla-cms/pull/45098#issuecomment-2708505806 ) are suggested.
Thank you.